service/handler_accounts.go

```diff
@@ -48,7 +48,14 @@ type AssertionRequest struct {
 // AccountsHandler is the API method to list the account assertions
 func AccountsHandler(w http.ResponseWriter, r *http.Request) {
 
-	accounts, err := datastore.Environ.DB.ListAccounts()
+	// Get the user from the JWT
+	username, err := checkUserPermissions(w, r)
```

Comment:
No - as I mentioned in the previous PR, I've now added a permissions check into that method. So the name now  makes sense.